src/tools/debugger/nodes/BreakpointPromiseRcvNode.java

```diff
@@ -0,0 +1,39 @@
+package tools.debugger.nodes;
```

Comment:
to refactor the BreakpointNode classes I think there is a problem because of the types of breakpoints. Indeed the BreakpointEnabling object is the one that makes the difference, but to be able to use only one BreakpointNode class I think that we will have to use a superclass and this will bring some changes because the most part of implementation related to breakpoints is specified by each type of breakpoint.